components/scream/src/share/io/scorpio_output.hpp

```diff
@@ -140,6 +167,8 @@ class AtmosphereOutput
   ekat::Comm                                  m_comm;
   std::shared_ptr<const FieldManager<Real>>   m_field_mgr;
   std::shared_ptr<const GridsManager>         m_grid_mgr;
+  grid_ptr_type                               m_grid;
```

Comment:
I'm confused by seeing both a grids manager and a grid. If you need just one grid, then there's no need to keep the grids manager around. OTOH, if you need more than one grid, then what's `m_grid`?